### ISSUE TITLE
[Android] Japan Date Format Wrong

### ISSUE THREAD

**Issue by crazypioneer** [henninghall/react-native-date-picker#176]
[Android] Japan Day Format Wrong

Now, showing like this:

> 2020年3月11日水曜日3月２０

Could you help me to change the format like this?

> 2020年3月11日

or

> 2020年3月11日（水）

```
<DatePicker
            mode='date'
            date={value}
            locale='ja'
            onDateChange={date => this.setState({value: date})}
/>

```
 - React Native version [0.61.4]
 - react-native-date-picker version [2.7.0]

**Comment by KenHoang16CDTH12** [comment]
I have the same error.

**Comment by henninghall** [comment]
Hi, have you tried the latest version?

**Comment by crazypioneer** [comment]
Thanks for your kind.
The last one updated 8 days ago...
I checked it 4 days ago...
Means, I 'm on the latest.

**Comment by henninghall** [comment]
You reported version 2.7.0 in the issue template. That's why I asked.

**Comment by ipk2015** [comment]
I fix the bug and create a pull request, plz review it. Thanks.

**Comment by henninghall** [comment]
Thanks a lot! 👍I will check it out shortly

**Comment by henninghall** [comment]
Released in v2.7.9

**Comment by crazypioneer** [comment]
@henninghall 
確認出来ました。
本当に嬉しいです！〜
ありがとうございます〜〜〜
